web/src/actions/admin.js

```diff
@@ -1,45 +1,19 @@
-import { notify } from './notification';
 import axios from '../util/api';
 
-export const ADMIN_CREATE_USER_REQUEST = Symbol(
-  'admin : create user : request'
-);
-export const ADMIN_CREATE_USER_SUCCESS = Symbol(
-  'admin : create user : success'
-);
-export const ADMIN_CREATE_USER_ERROR = Symbol('admin : create user : error');
+export const ADMIN_GET_USERS_REQUEST = Symbol('admin : get users : request');
+export const ADMIN_GET_USERS_SUCCESS = Symbol('admin : get users : success');
+export const ADMIN_GET_USERS_ERROR = Symbol('admin : get users : error');
```

Comment:
prettier is kind of chaotic. Sometimes it adds whitespace, then 3 months later it takes them back.

![](https://media.giphy.com/media/720g7C1jz13wI/giphy.gif)